.circleci/config.yml

```diff
@@ -212,6 +223,9 @@ workflows:
       - vscode:
           requires:
             - tarball
+      - wasm:
+          requires:
+            - tarball
```

Comment:
Oh I got this wrong. I assumed vscode didn't depend on tarball. Still given that it does this is the right outcome.